controllers/construct/mongodbstatefulset.go

```diff
@@ -307,11 +310,16 @@ func readinessProbeInit(volumeMount []corev1.VolumeMount) container.Modification
 
 func getMongoDBImage(version string) string {
 	repoUrl := os.Getenv(MongodbRepoUrl)
+	imageType := os.Getenv(MongoDBImageType)
```

Comment:
nit, can be in a follow-up:  wdyt about a helper function that returns the default if provided?
```
getEnvWithDefault(MongoDBImageType, DefaultImageType)
```

Reply:
Done